System.Web.Mvc.Expressions/System.Web.Mvc.Expressions.csproj

```diff
@@ -40,37 +40,32 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Web" />
     <Reference Include="System.Web.Helpers, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.AspNet.WebPages.3.2.3\lib\net45\System.Web.Helpers.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\packages\Microsoft.AspNet.WebPages.3.2.7\lib\net45\System.Web.Helpers.dll</HintPath>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.AspNet.Mvc.5.2.3\lib\net45\System.Web.Mvc.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="System.Web.Mvc, Version=5.2.7.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Would it be possible to not reference an explicit version here? So it'll work regardless of what version is actually installed?

Reply:
Thanks for looking into it. I am not familiar enough with the possible reference styles to answer that question. The other consideration is that even if Visual Studio will be able to locate the reference, when it builds, it's might still bind to the specific version located at compile time.

This SO answer seems to support my thinking: https://stackoverflow.com/questions/24022134/how-exactly-does-the-specific-version-property-of-an-assembly-reference-work-i, but I have not tested it myself.

Reply:
Actually it's best for a library to use the lowest possible version of it's dependencies and mark the dependency with `<SpecificVersion>False</SpecificVersion>` (as described in the SO answer @SvenAelterman refers to). So maybe it's better to downgrade to 5.0.0 (basically what @ivaylokenov did in this commit https://github.com/ivaylokenov/ASP.NET-MVC-Lambda-Expression-Helpers/commit/f56c65772a1b1ba278f6abeb36cdab638219183c#diff-64f0574f3b8754e57f68807d79fc91a8). About the green squiggly that's mentioned in issue #21: targeting 5.0.0 MVC and 3.0.0 Razor + WebPages packages doesn't generate it - tested it locally. The detailed build log shows that the 5.2.7 version of MVC is used:
```
1>  Unified primary reference "System.Web.Mvc, Version=5.2.7.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35".
1>      Using this version instead of original version "5.0.0.0" in "<BASE-PATH>\My-ASP.NET-MVC-Lambda-Expression-Helpers\System.Web.Mvc.Expressions\bin\Release\System.Web.Mvc.Expressions.dll" because of a binding redirect entry in the file "Web.config".
```